contracts/src/bridge/L1Oracle.sol

```diff
@@ -1,11 +1,11 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.4;
 
-import {Initializable} from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
 import {UUPSUpgradeable} from "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
 import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import {PausableUpgradeable} from "@openzeppelin/contracts-upgradeable/security/PausableUpgradeable.sol";
 
-contract L1Oracle is Initializable, UUPSUpgradeable, OwnableUpgradeable {
```

Comment:
why remove `Initializable`?

Reply:
`PausableUpgradeable` is already `Initializable`. I was having linearization errors in one spot (not here) and I thought it best to simplify since it's implicitly there. Can add back in here, I don't think it'll present compiler issues.

Reply:
sure just add it back so the entire codebase is consistent, all other contracts has `Initializable` with `PausableUpgradeable`